src/DatePicker/component/index.web.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import * as React from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+
+import type { Props } from './DatePickerTypes';
+
+type State = {
+  date: Date,
+};
+
+export default class WebDatePicker extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      date: props.date || new Date(),
+    };
+  }
+
+  setDefaultDate = () => {
+    this.setState({
+      date: new Date(),
+    });
+  };
+
+  handleConfirm = (date: Date) => {
+    const { onConfirm } = this.props;
+
+    this.setState({
+      date,
+    });
+
+    onConfirm(date);
+    this.setDefaultDate();
```

Comment:
What about removing this reset call? I think that for web more common behaviour is to let picker stay on last selected day.